SDM/posts/models.py

```diff
@@ -1,3 +1,4 @@
+from tkinter import CASCADE
```

Comment:
What's this for?

Reply:
Don't know how that got there must have been a wrongly typed code that brought that up sha.